src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Windows.cs

```diff
@@ -279,18 +283,40 @@ protected virtual void UpdateEmptyView()
 
 		protected virtual void UpdateItemsLayout()
 		{
-			ListViewBase.IsSynchronizedWithCurrentItem = false;
+			if (ListViewBase is FormsGridView gridView)
+			{
+				if (Layout is LinearItemsLayout linearItemsLayout)
+				{
+					gridView.Orientation = linearItemsLayout.Orientation == ItemsLayoutOrientation.Horizontal
+						? Orientation.Horizontal
+						: Orientation.Vertical;
 
-			FindScrollViewer(ListViewBase);
+					gridView.Span = 1;
 
-			_defaultHorizontalScrollVisibility = null;
-			_defaultVerticalScrollVisibility = null;
+					if (linearItemsLayout.ItemSpacing != _previousItemSpacing)
+					{
+						_previousItemSpacing = linearItemsLayout.ItemSpacing;
+						gridView.ItemContainerStyle = linearItemsLayout.GetItemContainerStyle();
+					}
+				}
 
-			UpdateItemTemplate();
```

Comment:
why is this removed? were is It being called now?

Reply:
With the PR changes nowhere, check [this comment](https://github.com/dotnet/maui/pull/14532/files#r1164070864) and let me know if you have any questions.